src/components/UI/Pagination/PaginationBox.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from "react";
+
+export interface PaginationBoxProps {
+  children: React.ReactNode;
+  disable?: boolean;
+  onClick: () => void;
+  pageNumber?: number;
+  currentPage?: number;
+  index: number;
+}
+
+export const PaginationBox: FunctionComponent<PaginationBoxProps> = ({
+  children,
+  disable,
+  onClick,
+  pageNumber,
+  currentPage,
+  index,
+}) => {
+  return (
+    <button
+      key={index}
+      className={`border-r border-solid border-grey-200 p-0 text-blue inline-block h-8 w-8 flex items-center justify-center hover:bg-grey-100 hover:text-grey focus:outline-none ${
+        currentPage === pageNumber && (currentPage || pageNumber) && "text-grey bg-grey-100"
```

Comment:
```
const active = currentPage === pageNumber && (currentPage || pageNumber) ? "text-grey bg-grey-100" : "";
const disabled = disable ? "bg-opacity-25 text-opacity-25 cursor-not-allowed" : "";
```
then in the className can use `${active} ${disabled}`, more readable i guess. also it seems like better to use ternary with empty string if condition not met, so it doesnt render undefined in the classname

Reply:
ok